### PR TITLE
UX Workflow to Submit Data to OEC

### LINKED ISSUE #25: As a provider who has verified my data are correct, I can submit my data to OEC.
When I have verified that my uploaded data are correct, I can click "Send to OEC" to submit my data to OEC.

Upon successful submission, I see the [screen for successful submission](https://app.abstract.com/share/5ea2033d-b4eb-4322-8a2a-1b601b77c8fc?collectionId=e2036f17-be42-40a0-9a0b-aa94c3efbc51&collectionLayerId=799d0ece-6864-465d-bbb3-eddce905deb4&present=true&preview=false&sha=c48a65163e8532e6de4b7594d18763f321e83306) that contains the contact email for the pilot, a .csv download of their enrollment data, and a link to upload another spreadsheet.

### PR DESCRIPTION
**Pull request by JordanGuinn**
Closes #25.

### Sibling PR
- https://github.com/ctoec/component-library/pull/43

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@ctoec/component-library": "^0.0.26",
+    "@ctoec/component-library": "https://github.com/ctoec/component-library#add-logo-with-check",
```
Comment (JordanGuinn) [inline client/package.json:6]: I'll undo this once right before merge.

File: client/src/components/CSVDownloadLink.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Button, TextWithIcon, ArrowDown as DownloadArrow } from '@ctoec/component-library';
+
+export const CSVDownloadLink = () => (
+  <Button
+    appearance="unstyled"
+    //  TODO: Have this actual download uploaded data, not the upload template
```
Comment (jlhogan) [inline client/src/components/CSVDownloadLink.tsx:7]: i believe @cailyncodes has an export data PR in flight so this should be solved by that !

File: client/src/components/CSVDownloadLink.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    //  We'll probably need to generate a CSV on the fly, since I don't think we're
+    //  holding uploaded files indefinitely?
+    href="/upload_template/ECE Data Collection Template.csv"
+    className="text-bold margin-bottom-3 display-block"
+    external
+    text={
+      <TextWithIcon
+        text="Download .csv"
+        Icon={DownloadArrow}
```
Comment (jlhogan) [inline client/src/components/CSVDownloadLink.tsx:16]: actual download arrow is in component lib, but not exported correctly -- comments on that other PR with more info. But i think you should use that svg here

File: client/src/components/CSVDownloadLink.tsx (hunk 2 above)
Comment (cailyncodes) [reply client/src/components/CSVDownloadLink.tsx:7]: Indeed! I had only been working on the backend part of it so will work off of this PR for the front end bits :)

**Pull request merged**